Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -550,6 +551,15 @@ TEST_CASE(test_jpeg_malformed_frame)
     }
 }
 
+TEST_CASE(test_jpeg2000_simple)
```

Comment:
This is a nitpick for the *previous* commit, but there is no way to leave a comment on a binary file.

Test images that aren't attached to a test case usually live in `/Base`. If an image is attached to a particular test case, it should probably go into the same commit as the test case itself.

Reply:
Sure, squashed them together.

(If the image is tricky to create, I like putting it in its own commit so I have some space to talk about the creation. Here it was easy to create though.)